packages/scratch-gui/package.json

```diff
@@ -46,7 +46,7 @@
     "i18n:push": "tx-push-src scratch-editor interface translations/en.json",
     "i18n:src": "rimraf ./translations/messages/src && babel src > tmp.js && rimraf tmp.js && build-i18n-src ./translations/messages/src ./translations/",
     "start": "webpack serve",
-    "test": "npm run test:lint && npm run test:unit && npm run test:integration",
+    "test": "npm run test:lint && npm run test:unit && npm run build && npm run test:integration",
```

Comment:
I'd prefer not to build as part of `npm run test` since `.github/workflows/ci.yml` builds in one job before it tests in a completely separate job.